test/specs/commonTests/isConformant.tsx

```diff
@@ -32,12 +33,19 @@ export default (Component, options: any = {}) => {
 
   const componentType = typeof Component
 
-  // This is added because of the FelaTheme wrapper and the component itself, because it is mounted
+  // This is added because the component is mounted
   const getComponent = wrapper => {
-    return wrapper
+    // FelaTheme wrapper and the component itself:
+    let component = wrapper
       .childAt(0)
       .childAt(0)
       .childAt(0)
+    if (component.type() === FocusZone) {
+      component = component // <FocusZone> wrap
+        .childAt(0) // <div> inside FocusZone wrap
+        .childAt(0) // the actual component
+    }
+    return component
```

Comment:
Do we have idea how will the Custom FocusZone be handled? Is it going to be completely different implementation for each custom zone, where in some cases there will be wrappers and in some other won't. I am a bit scared about how we would be able to provide the rendered component itself in those cases.

Reply:
Should we test here the mode wrap in this case as well?

Reply:
I think the idea of FocusZoneMode is still a little bit uncertain. The custom FocusZoneMode is that the consumer of Stardust would provide his own functions, meaning that it will actually never affect these unit tests. I think @jurokapsiar could provide more details? Should we modify this test to check FocusZoneMode?

Reply:
yes - custom means that the focus zone functionality is handled outside of Stardust. We have test (handlesAccessibility for menu-test) that checks if the component is correctly wrapped in the focus zone. The isConformant just skips FocusZone if it finds it. Currently we do not have tests that test only the behaviors - we just have tests that check, if the behaviors correctly modify particular components. It would be useful to 3 tests that just check if renderComponent uses FocusZone correctly - one for wrap, one for custom and one without FocusZone. Is this what you meant @mnajdova?

Reply:
I think that the custom mode is basically equivalent to disabling FocusZone. Is that right, @jurokapsiar? In that case, maybe we can completely eliminate the FocusZoneMode for now. 🤔

Reply:
@jurokapsiar you are right that's what I meant, explicitly handing having mode wrap, custom or no focus zone, but I was uncertain what is the expected result of having custom focus zone. If it all sums up to having this condition, then we are good to stay with this condition as is.

Reply:
Ok, let's leave it as it is for now. :)